src/chainparams.cpp

```diff
@@ -614,6 +623,15 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0020].nThresholdMin = 60; // 60% of 100
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0020].nFalloffCoeff = 5; // this corresponds to 10 periods
 
+        // Deployment of decreased proposal fee
+        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].bit = 7;
+        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nStartTime = 1625097600; // July 1st, 2021
+        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nTimeout = 1656633600; // July 1st, 2022
```

Comment:
same but a bit earlier:
```suggestion
        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nStartTime = 1632700800; // Sep 27th, 2021
        consensus.vDeployments[Consensus::DEPLOYMENT_GOV_FEE].nTimeout = 1664236800; // Sep 27th, 2022
```